social_navigation/Trajectory[] trajectories
